rockpi4.mk

```diff
@@ -195,6 +193,9 @@ rkdeveloptool-distclean:
 
 clean: rkdeveloptool-clean
 
+# In case wget fails mid-way
+.DELETE_ON_ERROR:
```

Comment:
This is a global setting, affecting all targets that can fail. I'm not sure where it's best to put it, but this close to the end of the file should make it hard to notice.

Reply:
> This is a global setting, affecting all targets that can fail.

Correct.

> I'm not sure where it's best to put it, but this close to the end of the file should make it hard to notice.

I can move it up